### PR TITLE
Russian Roulette

### LINKED ISSUE #4384: [New game]:  Russian Roulette
### 🎮 Game Request

- A simple Russian Roulette Game Where you test your luck by shooting bullets by Clicking a Button.
- The Gun has a 33% chance of shooting and if shot a shooting sound plays and you die and the death counter increases.

### Point down the features

Game points:
- A gun
- A button that when clicked shoots gun
- 33% chance of gun shooting
- when shot you die and death counter increases

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by Harshitmishra001**
## PR Description 📜

A simple Russian Roulette Game Where you test your luck by shooting bullets by Clicking a Button.
The Gun has a 33% chance of shooting and if shot a shooting sound plays and you die and the death counter increases.

Fixes #4384 

<hr>
 
## Mark the task you have completed ✅

- [✅ ] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [ ✅] I have performed a self-review of my own code or work.
- [ ✅] I have commented my code, particularly in hard-to-understand areas.
- [ ✅] My changes generates no new warnings.
- [ ✅] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [✅ ] I have added screenshot for website preview in assets/images 
- [ ✅] I have added entries for my game in main README.md
- [ ✅] I have added README.md in my folder 
- [✅ ] I have added working video of the game in README.md (optional)
- [✅ ] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸

Screenshot 2024-06-10 145820

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @Harshitmishra001 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

File: Games/Russian_Roulette/style.css
Comment (ishita-43) [inline Games/Russian_Roulette/style.css:1]: rename as your game folder name refer contribution guidelines

**Review by ishita-43 — CHANGES_REQUESTED** [review]

File: Games/Russian_Roulette/style.css
Comment (Harshitmishra001) [reply Games/Russian_Roulette/style.css:1]: Hey I am not understanding how I am supposed to name the folder?
The first letter is capital and _ is also used for spaces 
Files are also named correctly
Then what is the issue?

Comment (ishita-43) [reply Games/Russian_Roulette/style.css:1]: sorry i was referring your images in asset folder

**Comment by kunjgit** [comment]
Thank you  @Harshitmishra001 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request closed without merging**